app/views/groups/samples/index.turbo_stream.erb

```diff
@@ -0,0 +1,29 @@
+<%= turbo_stream.update "group_samples_list" do %>
+  <table class="mt-4 w-full text-sm text-left text-gray-500 dark:text-gray-400">
+    <thead class=" text-gray-700 bg-gray-50 dark:bg-gray-700 dark:text-gray-400 ">
+      <tr>
+        <th scope="col" class="px-6 py-3"><%= t("activerecord.attributes.sample.name") %></th>
+        <th scope="col" class="px-6 py-3"><%= t("activerecord.attributes.project.name") %></th>
+      </tr>
+    </thead>
+    <tbody id="group-samples-table-body">
+      <% @samples.each do |sample| %>
+        <tr
+          id="<%= dom_id(sample) %>"
+          class="bg-white border-b dark:bg-gray-800 dark:border-gray-700"
+        >
+          <td class="px-6 py-3"><%= sample.name %></td>
```

Comment:
I think the sample should link to the sample page (for the sample in the given project) and the project should link to the project page.

Reply:
I would also like to see the sample description like on the project > samples table

Reply:
Please see 96c612e8a69723dac36a48be26c23509e5eef35b.

Reply:
Thanks